packages/compass-connect/src/components/form/authentication/kerberos/cname-input.jsx

```diff
@@ -14,6 +14,7 @@ class CnameInput extends React.PureComponent {
 
   onCnameToggle() {
     Actions.onCnameToggle();
+    Actions.onConnectionFormChanged();
```

Comment:
What was happening before then :)?

Reply:
Well it just didn't ask you to save it :D It did save it eventually but it was bad UX.